src/Http/Controllers/AppointmentController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace LibreEHR\FHIR\Http\Controllers;
+
+use Illuminate\Http\Request;
+use LibreEHR\FHIR\Adapters\FHIRAppointmentAdapter;
+use Illuminate\Support\Facades\DB;
+use phpDocumentor\Reflection\DocBlock\Tags\Return_;
+
+class AppointmentController extends AbstractController
+{
+    public function __construct()
+    {
+        $this->adapter = new FHIRAppointmentAdapter;
+    }
+
+    public function index()
+    {
+
+    }
+
+    public function getSchedule()
+    {
+        $results = DB::select('select * from openemr_postcalendar_events');
+        $json = json_encode($results);
+
+        return $json;
+    }
+
+    public function addSlot( Request $request )
+    {
+        $start = $request->input('start');
```

Comment:
The request parameters will come in format compliment with FHIR API. Please look at example JSON in the FHIR documentation for Appointment resource. This controller implementation would not be able to accept FHIR request format. Please use the adapter layer to parse the request into a PHP FHIR Appointment model and store in the EMR database. See PatientCintroller and FHIR Patient Adapter for example.